### ISSUE TITLE
Failed to transform yaml input stream into object

### ISSUE THREAD

**Issue by Jinchul81** [kubernetes-client/java#229]
My system is built based on Kubernetes v1.9, client-java-parent-1.0.0-beta3
Unexpected exception occurred when loading yaml file which contains 'storage: 1Gi'. The root cause was that Quality class did not have a constructor to take a single string parameter. I was able to resolve the problem by adding the following code snippet. Let me provide a patch and request a PR.

A code snippet into Quantity.java
```
    public Quantity(final String value) {
        final Quantity quantity = fromString(value);
        this.number = quantity.getNumber();
        this.format = quantity.getFormat();
    }
```

A code snippet for YAML file loading.
```
    try {
      V1beta2StatefulSet body = yaml.loadAs(new FileReader(filePath), V1beta2StatefulSet.class);
```

An example of StatefulSet for YAML file.
```
  ...
  volumeClaimTemplates:
  - metadata:
      name: www
    spec:
      accessModes: [ "ReadWriteOnce" ]
      resources:
        requests:
          storage: 1Gi
      storageClassName: managed-nfs-storage
```

**Comment by Jinchul81** [comment]
A few questions before sending a PR.
1. Do you think the solution is promising?
2. Do I need to add a test? If so, where should I add an unit test?

**Comment by kondapally1989** [comment]
I think it has been fixed in https://github.com/kubernetes-client/java/pull/223

**Comment by brendandburns** [comment]
Yeah, I think this is fixed in #223, I'm going to cut another release, can you either test at head, or wait for the release and validate the fix? Re-open if it still occurs.

Thanks!
